tests/versioned/apollo-server/errors.test.js

```diff
@@ -9,7 +9,7 @@ const test = require('node:test')
 
 const { afterEach, setupCoreTest } = require('../../lib/test-tools')
 
-const errorsTests = require('../errors-tests')
+const errorsTests = require('./errors-tests')
```

Comment:
i don't see the need for this file. you can prob just move the `errors-tests` into here or make that file run the tests.